frontend/src/main/utils/dateUtils.js

```diff
@@ -6,4 +6,15 @@ const timestampToDate = (timestamp) => {
     return (date.getFullYear() + "-" + (padWithZero(date.getMonth()+1)) + "-" + padWithZero(date.getDate()));
 }
 
-export {timestampToDate, padWithZero};
\ No newline at end of file
+// Disabling stryker here because the returned value depends on the date the test is run
+// so a deterministic test is not possible
+// Stryker disable all
+const daysSinceTimestamp = (date) => {
+    var today = new Date();
+    var startingDate = new Date(date);
+    var timeDiff = Math.abs(today.getTime() - startingDate.getTime());
+    return Math.ceil(timeDiff / (1000 * 3600 * 24));
+}
+// Stryker enable all
```

Comment:
we can probably eliminate the need for a stryker disable here, so we can remove this; but if we don't, it should be

```suggestion
// Stryker restore all
```